### ISSUE TITLE
Acknowledge EAB copyright

### ISSUE THREAD

**Issue by moshemoshe137** [moshemoshe137/EAB_tools#11]
Add something like
```markdown
## Acknowledgments

This project is not affiliated with, endorsed by, or sponsored by EAB Global, Inc. 
"EAB Navigate" is a trademark of EAB Global, Inc. The use of the name "EAB Navigate" 
in this project is for identification purposes only and does not imply any 
association with EAB Global, Inc.

## Disclaimer

This independent project is developed for educational and informational purposes, 
to assist users in analyzing data they have legally exported from the EAB Navigate 
software. Users are responsible for ensuring their use of this tool complies with 
EAB's Terms of Use and any other applicable EAB policies. This project does not 
reproduce, distribute, or commercially exploit any EAB properties or services.


```
Written by [ChatGPT](https://chat.openai.com/share/a03d830e-5139-437f-b6f8-51ecaa76c9b3).

**Comment by moshemoshe137** [comment]
Perhaps add
```markdown
## Additional Notes

- This project's compatibility with EAB Navigate data relies on publicly available
data formats or legally exported data. It does not reverse engineer or derive
methods from EAB Navigate's proprietary technologies.

- For concerns about the use of the "EAB Navigate" trademark, please contact
[your contact information].

- Efforts have been made to avoid confusion or misrepresentation of this project's
affiliation with EAB Global, Inc., or its products, including EAB Navigate.

- Mention of EAB Navigate within this project does not imply endorsement by
EAB Global, Inc.
```

**Comment by moshemoshe137** [comment]
Closed by #21
